tests/python/twitter/common/reviewboard/test_reviewboard.py

```diff
@@ -0,0 +1,114 @@
+from twitter.common.reviewboard.reviewboard import ReviewBoardServer
+
+import mock
+from mock import call
+import pytest
+
+@mock.patch("twitter.common.reviewboard.reviewboard.ReviewBoardServer.http_request")
+def test_fetch_review_requests(mock_method):
+  mock_method.return_value = '''{
+    "stat" : "ok",
+    "review_requests" : [
+      { "id" : 123 }, { "id" : 456 }
+    ]
+  }'''
+  rbs = ReviewBoardServer('http://example.com', username='user', password='pass1')
+  assert len(rbs.fetch_review_requests()) == 2
+  mock_method.assert_called_once_with(
```

Comment:
mind moving this to:

```
assert mock_http_request.mock_calls == [mock.call( "api/review-requests/?start=0&max-results=25",
    None, None, {'Accept': 'application/json'}, method=None)]
```

We've found this helps make mocked calls much more explicit, and prevents typos in the assert calls.